pages/download.html

```diff
@@ -1,54 +1,63 @@
 <#include "incl/macros.html">
-<#def TITLE>Coq <#CURRENTVERSION></#def>
+<#def TITLE>Install Coq</#def>
 <#include "incl/header.html">
 
 <div class="framework">
-<div class="frameworklabel">The current version: Coq <#CURRENTVERSION>
+<div class="frameworklabel">The Coq platform
 </div>
 <div class="frameworkcontent">
 
-<p>For downloads (in particular to get installers for Windows and Mac OS, the PDF
-  manual, or a tarball of the sources), please go to
-  <a href="<#CURRENTVERSIONURL>">the release page on GitHub</a>.</p>
+<p>Coq has a rich ecosystem of external packages (libraries and
+plugins) that extend it and make it more powerful.  The Coq platform
+provides an easy way to install Coq and a consistent set of packages
+on Windows, macOS and many Linux distributions.</p>
+
+<p>Beginners are recommended to use one of the binary installers: we
+provide <a href="https://github.com/coq/coq/releases/latest">binary
+installers</a> for Windows and macOS as well as
+a <a href="https://snapcraft.io/coq-prover">Snap package</a>
+compatible with many Linux distributions.</p>
+
+<p>Experienced users are advised to run the scripts provided
+by <a href="https://github.com/coq/platform">the platform</a> to
+install from sources as this will allow them to further extend the
+resulting package set <a href="/opam-using.html">using opam</a>.</p>
```

Comment:
```suggestion
<p>Experienced users are advised to run the scripts provided
by <a href="https://github.com/coq/platform">the "Coq platform"</a> to
install from sources as this will allow them to further extend the
resulting package set <a href="/opam-using.html">using opam</a>.</p>
```

Seems like building Coq from sources and building with coq/platform are separate options that should be described separately.  This confused me for a moment because "Coq platform" is a very generic term that you're using here as a defined term.

Reply:
OK to add the Coq qualifier, but why the quotes (especially in a paragraph about the Coq platform)?

Reply:
Because "the Java platform", "the Python platform", etc. is generic phrasing.  If I build Coq from sources, I think I've built the Coq platform, whereas here IIUC it's being used as a defined term  (hence the quotes) to refer to what you get from building from sources _plus_ a bunch of additional libraries.  Might help to make this distinction more explicit?  Does that make any sense?

Reply:
I think making "Coq platform" a link will do just as well as adding quotes.